doc/modules/mod_event_pusher_http.md

```diff
@@ -0,0 +1,59 @@
+## Module description
+
+This module is a backend of [mod_event_pusher] that enables forwarding certain events (messages, presence, etc.) via HTTP to external services such as push (by mobile, email or SMS), big data, or analytics services.
+
+### How it works
+
+The module hooks on all packets sent by connected users.
```

Comment:
The module hooks on all packets sent by the connected users?

Reply:
Nah, all connected users.